model/project_ref.go

```diff
@@ -744,22 +778,89 @@ func FindOneProjectRefByRepoAndBranchWithPRTesting(owner, repo, branch string) (
 			return &p, nil
 		}
 	}
+	if len(projectRefs) > 0 {
+		grip.Debug(message.Fields{
+			"source":  "find project ref for PR testing",
+			"message": "project ref enabled but pr testing not enabled",
+			"owner":   owner,
+			"repo":    repo,
+			"branch":  branch,
+		})
+		return nil, nil
+	}
 
-	grip.Debug(message.Fields{
-		"message": "no matching project ref with pr testing enabled",
-		"owner":   owner,
-		"repo":    repo,
-		"branch":  branch,
-	})
-	return nil, nil
+	// if no projects are enabled, check if the repo has PR testing enabled, in which case we can use a disabled/hidden project.
+	repoRef, err := FindRepoRefByOwnerAndRepo(owner, repo)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error finding merged repo refs for repo '%s/%s'", owner, repo)
+	}
+	if repoRef == nil || !repoRef.IsEnabled() || !repoRef.IsPRTestingEnabled() || repoRef.RemotePath == "" {
+		grip.Debug(message.Fields{
+			"source":  "find project ref for PR testing",
+			"message": "repo ref not configured for PR testing untracked branches",
+			"owner":   owner,
+			"repo":    repo,
+			"branch":  branch,
+		})
+		return nil, nil
+	}
+
+	projectRefs, err = FindAllMergedProjectRefByRepoAndBranch(owner, repo, branch)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error finding merged all project refs for repo '%s/%s' with branch '%s'",
+			owner, repo, branch)
+	}
+
+	// use disabled project if it exists, otherwise use hidden.
+	var hiddenProject *ProjectRef
+	for i, p := range projectRefs {
+		if !p.IsEnabled() && !p.IsHidden() {
+			return &p, nil
+		}
+		if p.IsHidden() {
+			hiddenProject = &projectRefs[i]
```

Comment:
An argument for excluding disabled/hidden projects from this is that if I specifically don't want pr testing (or anything that evergreen does) for this branch, I might either hide or disable it. And I think the inverse is true as well - if I hide or disable a branch, I'd expect that evergreen doesn't track anything from it

Reply:
Hidden projects are used internally to denote "untracked branches", so that's not something the user should even know about. The argument for disabled is reasonable.

Reply:
Even if an admin hid the project though, I'd think that we shouldn't run tasks for the hidden project, since hiding it sort of means "this doesn't exist" except that we don't want to break historical links to it

Reply:
No, what I mean is that admins don't hide projects: these are projects we create so that patches created for untracked branches still have a projectId associated for them. This is why at line 832, if we don't have a hiddenProject, we create a new one and insert it. Feel free to DM me if you need more clarification.

Reply:
The whole purpose of this ticket is to allow PR testing for untracked branches, i.e. "projects that don't exist"